hack/chains/README.md

```diff
@@ -33,10 +33,6 @@ demonstrate chains.)
     cd hack/chains
     ./create-signing-secret.sh
 
-### Apply some CA cert related hacks to make SSL work for the internal registry
-
-    ./trust-local-cert.sh
-
```

Comment:
I reworded this in #188 FYI so we might get a conflict. See if you like my change there and if it makes it worthwhile to keep mentioning this script in the readme.